g3doc/quick_reference.md

```diff
@@ -857,6 +857,23 @@ false is zero, true has all bits set:
     these operations. This op is for situations where the inputs are known to be
     mutually exclusive.
 
+*   <code>M **SetOnlyFirst**(M m)</code>: returns mask that has only lane `k`
```

Comment:
I mis-parsed this sentence. Maybe it would be more clear to say something like this?

If none of `m[i]` are true, returns all-true. Otherwise, only lane k is true, where k = FindKnownFirstTrue(m). In other words, sets to false any lanes with index greater than the first true lane, if it exists.

Reply:
I have updated the documentation of SetOnlyFirst and SetBeforeFirst and pushed the changes up to the repository.